package.json

```diff
@@ -78,38 +62,6 @@
     "cozy-scripts": "0.4.1",
     "cozy-ui": "7.0.3",
     "preact": "8.2.7",
-    "preact-compat": "3.17.0",
-    "react-redux": "5.0.6",
-    "react-router-dom": "4.2.2",
-    "redux": "3.7.2",
-    "redux-raven-middleware": "1.2.0",
-    "redux-thunk": "2.2.0"
-  },
-  "jest": {
```

Comment:
It may be better to keep this jest configuration if you want to write jest tests IMHO

Reply:
@enguerran create a specific config file in https://github.com/cozy/cozy-contacts/commit/f05c82898558edd1939552e519c9123a9ae5bde7 but it's not in the final diff, not sure why..?

Reply:
Everything is here: https://github.com/cozy/cozy-contacts/pull/8/files#diff-2d0cd5d10b9604941c38c6aac608178a

I remove the config because 0 config do not mean that it is already configured in a very opinionated way. This actual jest configuration let actual tests to pass. I don't want to write a test and discover that the cargo-culted configuration does not fit my needs. So I KISS the configuration.

> 0 config ain't cargo-culted config

Reply:
BTW

```
$ jest
 PASS  src/components/__tests__/App.spec.js
  App component only
    ✓ should be mounted correctly (22ms)
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   1 passed, 1 total
Time:        2.216s
Ran all test suites.
The command "yarn test" exited with 0.
```
link: https://travis-ci.org/cozy/cozy-contacts/builds/329945183#L690-L702